src/main/java/com/ldtteam/structurize/util/BlockChangeData.java

```diff
@@ -0,0 +1,107 @@
+package com.ldtteam.structurize.util;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.tileentity.TileEntity;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Class which preserves a state change of a block and TE
+ */
+public class BlockChangeData
```

Comment:
honestly could be a record

Reply:
java 8 :)

Reply:
well, someone renamed branched to latest, guess I cant guess the version from that :D